player/lua/osc.lua

```diff
@@ -2205,8 +2208,8 @@ end
 
 function reset_margins()
     if state.using_video_margins then
-        for _, opt in ipairs(margins_opts) do
```

Comment:
this is one of the shadow/redefinition warnings, right?
I'd just exclude those.

Reply:
yes, it shadows `opt`. I think it is very easy to miss, I'd like to keep it.